pkg/store/storepb/rpc.proto

```diff
@@ -0,0 +1,59 @@
+syntax = "proto3";
+package promlts;
+
+import "types.proto";
+import "google/protobuf/timestamp.proto";
+import "google/api/annotations.proto";
+import "gogoproto/gogo.proto";
+
+option go_package = "storepb";
+
+option (gogoproto.sizer_all) = true;
+option (gogoproto.marshaler_all) = true;
+option (gogoproto.unmarshaler_all) = true;
+option (gogoproto.goproto_getters_all) = false;
+
+service Store {
+  rpc Series(SeriesRequest) returns (SeriesResponse) {
+    option (google.api.http) = {
+        post: "/v1/series"
+    };
+  }
+
+  rpc LabelNames(LabelNamesRequest) returns (LabelNamesResponse) {
+    option (google.api.http) = {
+      post: "/v1/labels"
+    };
+  }
+
+  rpc LabelValues(LabelValuesRequest) returns (LabelValuesResponse) {
+    option (google.api.http) = {
+      post: "/v1/labels"
+    };
+  }
+}
+
+message SeriesRequest {
+  google.protobuf.Timestamp min_time = 1 [(gogoproto.stdtime) = true];
+  google.protobuf.Timestamp max_time = 2 [(gogoproto.stdtime) = true];
```

Comment:
Samples in TSDB have an `int64` timestamp. Interpretation of that value is up the clients. I suppose the question here is whether we want this to be a wall time object or just treat it as opaque `int64`s as well.

I'd lean towards changing to `int64`, especially since this is a pure data storage API and not for an end user of Prometheus, where the values are mapped to wall time.

But interested in more opinions first.

Reply:
I don't mind either.. I am fine moving to raw int64. However let's comment on that what it stores (from the TSDB perspective). I think we will operating on TSDB format anyway, so having time.Time is not that useful (?)